### PR TITLE
last changes to html, css, and @media made

### PR DESCRIPTION
**Pull request by lindalejiete**
1. footer height is now 100% not 40vh, so it doesn't break down on different screen sizes;
2. Section destination @media breakpoint is at 1100px, so the section doesn't break down;
3.  Changed font of accent from Artemis to Josefin Sans, font-weight used in project is 100;

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,30 +14,30 @@
     <section id="destinations" class="section-destinations">
         <div class="destinations-wrapper">
             <div class="destinations-textbox">
-                <h1>How to choose destination?</h1>
+                <h1>HOW TO CHOOSE DESTINATION?</h1>
```
Comment (KristianaTR) [inline index.html:111]: This could be done in css with text-transform: uppercase;

File: styles.css [hunk 2]
```diff
@@ -169,28 +171,38 @@ body {
 
 footer {
   color: var(--color-isabelline);
-  height: 40vh;
+  height:  100%;
```
Comment (KristianaTR) [inline styles.css:173]: I think you could leave vh --> 100vh since all section height is set to vh

File: index.html (hunk 1 above)
Comment (lindalejiete) [reply index.html:111]: good to know! will keep in mind

**Review by KristianaTR — APPROVED** [review]

File: styles.css (hunk 2 above)
Comment (lindalejiete) [reply styles.css:173]: for Contact us height 100vh is way too much, there is not so much content

Comment (lindalejiete) [reply styles.css:173]: Leave it like this

**Pull request merged**